.editorconfig

```diff
@@ -0,0 +1,19 @@
+# EditorConfig is awesome: http://EditorConfig.org
+
+root = true
+
+[*]
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.{js,coffee,html,less,css,scss,json}]
+indent_style = tab
```

Comment:
Nitpick, but I think most of the JS world uses 2 space indentation; how strongly would you want tabs here?